.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "FFmpeg"]
+	path = FFmpeg
+	url = https://github.com/FFmpeg/FFmpeg.git
+	branch = release/6.1
```

Comment:
FYI The submodule module commit you commited is this one:
https://github.com/FFmpeg/FFmpeg/commits/f309408874a5d6ccc77063631fa78b8fb59937a0/

that is different from the 6.1 release commit: https://github.com/FFmpeg/FFmpeg/commits/n6.1/

Can you please update it?

Reply:
Thanks, moved to `tags/n6.1` which points to the commit `d4ff0020b40b524a490cf62eccbd3a318f4c0e58`.